lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -241,9 +250,138 @@
+            </div>
+        </div>
+        <div class="section_specific">
+          <div class="email-list-container"
+               data-label ="Filter On"
+               data-selections="Opened <>
+                                Not Opened">
+          </div>
+        </span>
+    </div>
+        <div class="addingQuery">
+            <span id="incompleteMessage"></span>
+            <input type="button" name="addQuery" id="addQuery" value="Add Query" align="right">
+         </div>
+    </div>
+    <div class="email-list-widget">
+    <div class="email-list">
+      <table id="queryTable">
+          <caption class="tableCaption">Active Queries</caption>
+        <thead>
+          <tr>
+              <td class="label">Inclusion</td>
+              <td class="label">Type</td>
+              <td class="label">Specification</td>
+              <td class="label">Criteria</td>
+              <td class="label">Status</td>
+              <td class="label"></td>
+          </tr>
+        </thead>
+        <tbody id="queryTableBody"></tbody>
+      </table>
+      <br/>
+      <div class="bottom-email">
+          <input type="button" name="getcsv" id="getcsv" value="Get CSV" data-endpoint="${ section_data['get_all_students'] }" data-csv="true" class="disabled">
+          <input type="button" name="savequery" id="savequery" value="Save Query" data-endpoint="${ section_data['save_query'] }" data-csv="true" class="disabled">
+          <div id="estimated">Approx 0 students selected</div>
+          <input type="button" name="getest" id="getest" value="Update Students Estimate" data-endpoint="${ section_data['get_all_students'] }" >
+          <input type="button" name="startover" id="startover" value="Clear Active Queries">
+      </div>
+      <table>
+          <caption class="tableCaption">Saved Queries</caption>
+        <thead>
+          <tr>
+              <td class="label">Date Saved</td>
+              <td class="label">Query</td>
+              <td class="label"></td>
+              <td class="label"></td>
+          </tr>
+        </thead>
+        <tbody id="savedQueriesTable" data-endpoint="${ section_data['get_saved_queries'] }"></tbody>
+
+      </table>
+        <table>
+            <tbody id="invisibleQueriesStorage" data-endpoint="${ section_data['get_saved_queries'] }"></tbody>
+        </table>
+       <br/>
+        <br/>
+        <div>
+        <h3>Example:</h3>
+        <div>AND, Problem 1, Opened</div>
+        <div>AND, Problem 2, Not Completed</div>
+        <div>NOT, Problem 3, Opened</div>
+        <div>OR, Problem 2, Not Opened</div>
+        <br/>
+        <div>Returns: Students who have opened Problem 1 and not completed Problem 2, but have not opened Problem 3 in
+            addition to students who have not opened Problem 4<br/><br/>
+            The intersection of <i>AND</i> queries is taken.<br/>
+            The union of  <i>NOT</i> queries is taken <br/>
```

Comment:
Minor point: In HTML, any consecutive whitespace is squashed into a single whitespace, so you could remove one of the duplicated spaces here (preceeding the `<i>NOT</i>`) and on the following line.